bin/picca_export.py

```diff
@@ -240,6 +241,44 @@ def main(cmdargs):
         'R-parallel', 'R-transverse', 'Redshift', 'Correlation',
         'Covariance matrix', 'Distortion matrix', 'Number of pairs'
     ]
+
+    # Check if we need blinding and apply it
+    if 'BLIND' in data_name or blinding != 'none':
+        if blinding != 'corr_yshift':
+            print('Only strategy E called "corr_yshift" is allowed for now.'
+                  ' This will be applied automatilly.')
+
+        # Figure out what which correlation it is
+        if len(xi) == 50:
```

Comment:
I would change the order of these ifs. Otherwise, there could be some binning raising an AssertionError instead of a ValueError with the correct message. I would do something like
```
if 'qso' in qrgs.data.lower():
    if len(xi) != 100:
        raise ValueError('Unknown binning. Cannot blind. Please raise an issue'
                                    ' or contact picca developers.')
    if 'lyb' in args.data.lower():
        blind_corr = 'qsoxlyb'
    else:
        blind_corr = 'qsoxlya'
else:
    if len(xi) != 50:
        raise ValueError('Unknown binning. Cannot blind. Please raise an issue'
                                    ' or contact picca developers.'))
    if 'lyb' in args.data.lower():
        blind_corr = 'lyaxlyb'
    else:
        blind_corr = 'lyaxlya'
```

Reply:
Yes that could work, but also add it for the qsoxlyb

Reply:
What I am worried is that at some point we might want to double the binning in r_par o r_trans thus having 100 pixels and then entering the wrong branch

Reply:
As I mentioned on Slack, we could add --blind-corr-type as an option to picca_export.py that would specify the correlation type. It would only be required when running on blind data.